include/RAJA/policy/openmp/target_reduce.hpp

```diff
@@ -48,6 +48,59 @@ namespace RAJA
 namespace omp
 {
 
+template <typename T>
+struct sum {
+  static constexpr T identity = T(0);
+  RAJA_HOST_DEVICE RAJA_INLINE
+  void operator()(T &val, const T v) { val += v; }
+};
+
```

Comment:
What was wrong with using `RAJA::reduce::sum` and others? We now have multiple implementations for the same reduction functions. Additionally, the new function objects aren't marked within a `#pragma omp declare target` block.

Reply:
Thanks for the heads up Will. I was actually thinking about your second point, and will get those marked, though to be fair reduction unit test passed without it.

Reply:
ON your first point there is a nuanced issue with the other RAJA::reduce::sum operator
which errors out with no matching constructor (for 2nd argument, i.e. identity), hence the reason I brought in some older operators.

Reply:
Ok, took care of the redundant sum,min,max. However minmaxloc didn't change as it's a stretch to utilize the ValueLoc  class.

Reply:
Ah okay. I’ll look into seeing if there’s a way to get it to work.